project/ReflectiveCodeGen.scala

```diff
@@ -6,9 +6,10 @@ import ProtocPlugin.autoImport.PB
 import akka.grpc.sbt.AkkaGrpcPlugin
 import akka.grpc.sbt.AkkaGrpcPlugin.autoImport._
 import protocbridge.{ Artifact, SandboxedJvmGenerator }
-import sbt.internal.inc.classpath.ClasspathUtilities
+import sbt.internal.inc.classpath.{ ClasspathUtil, ClasspathUtilities }
```

Comment:
:thinking:

Reply:
Fixed and split out to https://github.com/lightbend/akkaserverless-java-sdk/pull/707.